README.md

```diff
@@ -27,52 +27,52 @@ struct PlayerResolver: Resolver {
 
-Settler solves these problems by replacing your Factory with a **Resolver**. With a Resolver, you can break your Factory up into a flat collection of functions, each of which has a single responsibility: to build an object. A Resolver can be divided into many separate extensions that are not required to be co-located. It may actually be helpful to co-locate a Resolver extension with the object(s) it is responsible for creating, for example:
+Settler solves these problems by replacing factories with resolvers. With a `Resolver`, you can flatten your factory into a collection of functions, each of which has a single responsibility: to build an object. A resolver can be divided into many separate extensions that are not required to be co-located. It may actually be helpful to co-locate a Resolver extension with the object(s) it is responsible for creating, for example:
 
 ```swift
-class MyAPIService {
-    // ...
+final class MyAPIService {
+    // …
 }
 
 extension MyResolver {
-    func resolveAPIService(...) -> Key.APIService {
-        MyAPIService(...)
+    func resolveAPIService(…) -> Key.APIService {
+        MyAPIService(…)
     }
 }
 ```
 
 ## Settler as a methodology
 
-At this point, you might be asking yourself, "Can't I do all of this on my own?" The answer is, "Absolutely!" Once you've completely built and validated your Resolver and generated your Resolver output function, what you're left with is plain ol' Swift code. This is code you could have written yourself without the help of Settler. You could even remove Settler from your project completely at this point and your Resolver would continue to function properly. But if you choose to keep Settler as an integrated part of your build pipeline, what you'll get is what you had all along while building your Resolver: compiler-level enforcement of the _Settler methodology_.
+At this point, you might be asking yourself, “Can’t I do all of this on my own?” The answer is, “Absolutely!” Once you've built and validated your Resolver and generated your Resolver output function, what you’re left with is plain ol’ Swift code. This is code you could have written yourself without the help of Settler. You could even remove Settler from your project at this point and your resolver would continue to function properly. But if you choose to keep Settler as an integrated part of your build pipeline, what you’ll get is what you had all along while building your Resolver: compiler-level enforcement of the _Settler methodology_.
```

Comment:
Looks like there are a few examples of big "R" Resolvers in here. Should these be changed to little "r" as well?